### PR TITLE
Just swooping in to make events more event-y

### PR DESCRIPTION
**Pull request by omgmog**
This change:
- deletes some of the boilerplate from the 11ty netlify template
- makes events in to things that have pages
- makes the index D Y N A M I C

Now you'll be able to add the youtube video to an event by updating the "stream" frontmatter. This will then automatically be rendered as an iframe in the page :boom:

### CODE REVIEW

File: src/events/2021-01.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+   twitter: spikeheap
+   what: PR guidelines
+ - who: Ruth
+   twitter: Rumyra
+   what: MDN contributions
+ - who: Ben
+   twitter: benjaminbenben
+   what: Dev on Github
+ - who: Gabor
+   twitter: javorsky
```
Comment (javorszky) [inline src/events/2021-01.md]:
```suggestion
   twitter: javorszky
```
The z gets left out every time 🤣

**Review by javorszky — CHANGES_REQUESTED** [review]
a typo :)

**Comment by omgmog** [comment]
Oh also @javorszky this also does the same thing as your #5 in 980378d53ddbc1ffe96f74cf2b25c8fb51dd6648

**Review by javorszky — APPROVED** [review]

**Comment by javorszky** [comment]
> Oh also @javorszky this also does the same thing as your #5 in [980378d](https://github.com/jsoxford/jsnotoxfordsite/commit/980378d53ddbc1ffe96f74cf2b25c8fb51dd6648)

Nice! Closing #5 then

**Comment by omgmog** [comment]
Just need somebody who can merge this then

**Pull request merged**